Makefile

```diff
@@ -21,7 +21,8 @@ endif
 TEST_FORMAT = short-verbose
 endif
 
-PIPELINE_VERSION = 0.65.0
+PIPELINE_VERSION = instance-storage-api
```

Comment:
merge only after released

Reply:
yes, it was set to 0.65.1 but for some reason last commit was to picked up by the PR, anyway I've updated now.